docs/ref/models/expressions.txt

```diff
@@ -181,7 +181,8 @@ instance and will be applied on each :meth:`~Model.save()`. For example::
     reporter.save()
 
 ``stories_filed`` will be updated twice in this case. If it's initially ``1``,
-the final value will be ``3``.
+the final value will be ``3``. This persistence can be avoided simply by
+reloading the model object after saving it.
```

Comment:
It would be helpful to be explicit about how to reload the model, for example by adding "using `reload_from_db()`.

Reply:
I wasn't sure to include it since the doc already explains how to reload model objects ([somewhere above](https://docs.djangoproject.com/en/2.2/ref/models/expressions/#f-expressions)), but I guess you're right, it's better to mention `refresh_from_db()`.